### PR TITLE
Created the folder structure for the app and added redux

### PR DESCRIPTION
**Pull request by rkreddyy**

### CODE REVIEW

File: TrainingApp/src/components/Button/style.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+export default StyleSheet.create({
+  container: {
+    width: '100%',
+    height: 64,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    paddingHorizontal: 16,
+    backgroundColor: 'grey',
```
Comment (sharad-epam) [inline TrainingApp/src/components/Button/style.js]: use #hex  or Rgb codes

File: TrainingApp/src/components/Button/style.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  },
+
+  title: {
+    textTransform: 'uppercase',
+  },
+
+  containerLoading: {
+    backgroundColor: 'black',
```
Comment (sharad-epam) [inline TrainingApp/src/components/Button/style.js]: same here

File: TrainingApp/src/screens/Profile/Profile.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { View, Text } from "react-native";
+
+const Profile = () => {
+  return (
+    <View style={{ flex: 1, alignItems: "center", justifyContent: "center" }}>
```
Comment (sharad-epam) [inline TrainingApp/src/screens/Profile/Profile.js]: get these from styles file

File: TrainingApp/src/components/ProductGridItem/style.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import { StyleSheet } from 'react-native'
+
+export default StyleSheet.create({
+  container: {
+    paddingVertical: 16,
+    paddingHorizontal: 8,
+
+    backgroundColor: 'white',
```
Comment (sharad-epam) [inline TrainingApp/src/components/ProductGridItem/style.js]: use hex or rgb here

File: TrainingApp/src/App.js [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+import {
+  Header,
+  LearnMoreLinks,
+  Colors,
+  DebugInstructions,
+  ReloadInstructions,
+} from 'react-native/Libraries/NewAppScreen';
+
+// import { Provider } from 'react-redux';
+// import store from './store';
```
Comment (sharad-epam) [inline TrainingApp/src/App.js]: remove commented code

File: TrainingApp/src/components/Button/style.js (hunk 2 above)
Comment (rkreddyy) [reply TrainingApp/src/components/Button/style.js]: made required changes

**Pull request merged**